src/check_ast.rs

```diff
@@ -1326,6 +1326,12 @@ where
                         self.locate_check(Range::from_located(expr)),
                     ));
                 }
+
+                if self.settings.enabled.contains(&CheckCode::B015) {
+                    if let Some(parent) = self.parents.last() {
+                        flake8_bugbear::plugins::useless_comparison(self, expr, parent);
```

Comment:
```suggestion
                        println!("{:?}", expr);
                        flake8_bugbear::plugins::useless_comparison(self, expr, parent);
```

prints out:

```
Located {
  location: Location { row: 3, column: 2 },
  end_location: Some(Location { row: 3, column: 6 }),
  custom: (),
  node: Compare {
    left: Located { location: Location { row: 3, column: 0 },
    end_location: Some(Location { row: 3, column: 1 }),
    custom: (),
    node: Constant { value: Int(1), kind: None } },
    ops: [Eq], comparators: [Located { location: Location { row: 3, column: 5 }, end_location: Some(Location { row: 3, column: 6 }), custom: (), node: Constant { value: Int(1), kind: None } }] } }
```
```

Reply:
Could you use the start location of `node.left` for now?

Reply:
Got it. Made the change.